### ISSUE TITLE
Add a method to SLCosmo to read in cosmological parameter info from a .ini file.

### ISSUE THREAD

**Issue by jchiang87** [DarkEnergyScienceCollaboration/SLCosmo#13]
This info would include the truth and prior information, the latter for sampling the prior distribution.  It might be useful if the .ini file followed an existing format, say from cosmosis or class.

**Comment by drphilmarshall** [comment]
Agreed. This could be important when using this code in the TDC2, because the Good Teams definitely won't know the true value of H0 and they may not have been given the H0 prior either. However, for ease of use there probably should still be a default H0 prior. I'll think about this.
